tests/unit/lms/views/api/lti_test.py

```diff
@@ -158,12 +165,5 @@ def lti_outcomes_client(pyramid_config):
 @pytest.fixture(autouse=True)
 def ai_getter(pyramid_config):
     svc = mock.create_autospec(ApplicationInstanceGetter, instance=True, spec_set=True)
-
-    def shared_secret(consumer_key):
-        if consumer_key != "TEST_OAUTH_CONSUMER_KEY":
-            raise Exception("Incorrect consumer key")
-        return "oauth-secret"
-
-    svc.shared_secret.side_effect = shared_secret
```

Comment:
The `raise Exception("Incorrect consumer key")` (apparently used instead of an assertion) was never being hit which was reducing our coverage score. I've replaced this with explicit `assert_called_once_with(...)`'s in the relevant tests, which makes this fixture simpler, increases coverage/means we don't have dead test fixture code, and makes the tests easy to understand and less entangled with each other